src/Sylius/Behat/Context/Setup/CurrencyContext.php

```diff
@@ -93,11 +93,17 @@ public function theStoreHasCurrency($currencyCode)
 
     /**
      * @Given the store has currency :currencyCode, :secondCurrencyCode
+     * @Given the store has currency :currencyCode and :secondCurrencyCode
+     * @Given the store has currency :currencyCode, :secondCurrencyCode and :thirdCurrencyCode
      */
-    public function theStoreHasCurrencyAnd($currencyCode, $secondCurrencyCode)
+    public function theStoreHasCurrencyAnd($currencyCode, $secondCurrencyCode, $thirdCurrencyCode = null)
```

Comment:
Unfortunately, Variadic extension does not work well with turnip notation... I will have to fix it...